ui/templatetags/render_bundle.py

```diff
@@ -46,9 +55,46 @@ def _get_bundle(request, bundle_name):
 def render_bundle(context, bundle_name):
     """
     Render the script tags for a Webpack bundle
+
+    We use this instead of webpack_loader.templatetags.webpack_loader.render_bundle because we want to substitute
+    a dynamic URL for webpack dev environments. Maybe in the future we should refactor to use publicPath
+    instead for this.
+
+    Args:
+        context (dict): The context for rendering the template (includes request)
+        bundle_name (str): The name of the bundle to render
+
+    Returns:
+        django.utils.safestring.SafeText: The tags for JS and CSS
     """
     try:
-        return render_as_tags(_get_bundle(context['request'], bundle_name), '')
+        bundle = _get_bundle(context['request'], bundle_name)
+        return _render_tags(bundle)
     except OSError:
         # webpack-stats.json doesn't exist
-        return ''
+        return mark_safe('')
+
+
+def _render_tags(bundle):
+    """
+    Outputs tags for template rendering.
+    Adapted from webpack_loader.utils.get_as_tags and webpack_loader.templatetags.webpack_loader.
+
+    Args:
+        bundle (iterable of dict): The information about a webpack bundle
+
+    Returns:
+        django.utils.safestring.SafeText: HTML for rendering bundles
+    """
+
+    tags = []
+    for chunk in bundle:
+        if chunk['name'].endswith(('.js', '.js.gz')):
+            tags.append((
+                '<script type="text/javascript" src="{}" ></script>'
+            ).format(chunk['url']))
+        elif chunk['name'].endswith(('.css', '.css.gz')):
```

Comment:
Maybe add a unit test for this block by adding a `.css` file to `FAKE_COMMON_BUNDLE` in `render_bundle_test.py`

Reply:
Sounds good